test/e2e/case13_templatization_test.go

```diff
@@ -482,7 +482,7 @@ var _ = Describe("Test templatization", Ordered, func() {
 					case13PruneTmpErr+"-configmap", "default", true, defaultTimeoutSeconds)
 
 				return configmap
-			}, defaultTimeoutSeconds, 1).ShouldNot(BeNil())
```

Comment:
consistently timeout

Reply:
What do you mean?

I think this change is good - we don't need to check that the configmap is still there for 60 (or more) seconds, 30 should be fine. But I don't understand why it would be timing out right now?

Reply:
It doesn't any issue but I tried to reduce e2e test time. I think 60 secs is meaningless. Just suggestion. It took over 20mins before but now less than 15 mins